hilti/src/rt/tests/util.cc

```diff
@@ -362,23 +364,49 @@ TEST_CASE("map_tuple") {
     CHECK_EQ(input, std::make_tuple(2u, 2L, std::string("aa")));
 }
 
-TEST_CASE("resource_usage") {
-    auto ru = resource_usage();
+namespace hilti::rt {
+std::ostream& operator<<(std::ostream& stream, const ResourceUsage& ru) {
```

Comment:
are you using/testing this?

Reply:
I used this in development for printing `ResourceUsage` objects from doctest as otherwise it would use its default formatter (which just outputs `{?}`). I removed this for now.